opencti-platform/opencti-front/tests_e2e/model/menu/leftBar.pageModel.ts

```diff
@@ -0,0 +1,17 @@
+import { Page } from "@playwright/test";
+import { expect } from '../../fixtures/baseFixtures';
+
+export class LeftBarPage {
+  constructor(private page:Page) {
+  }
+
+  async clickOnMenu(menuName: string, pageName: string) {
+    await this.page.getByRole('menuitem', { name: menuName }).click();
+    await this.page.getByRole('menuitem', { name: pageName }).click();
+  }
```

Comment:
```suggestion
  async clickOnMenu(menuItem: string, subMenuItem?: string) {
    await this.page.getByRole('menuitem', { name: menuItem}).click();
    if (subMenuItem) {
      await this.page.getByRole('menuitem', { name: subMenuItem }).click();
    }
  }
```
By so we can use this function in `navigation.spec.ts` instead of calling directly `page.getRole`, just to simplify the readability of the test, what do you think?

Reply:
Done